packages/migrate/commonsChunkPlugin/__snapshots__/commonsChunkPlugin.test.js.snap

```diff
@@ -0,0 +1,316 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-0" data 1`] = `
+"module.exports = {
+    entry: {
+		app: './src/app.js',
+		vendor: './src/vendors.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                app: {
+                    name: 'app',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 2,
+                    test: 'app'
+                },
+
+                vendor: {
+                    name: 'vendor',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 2,
+                    test: 'vendor'
+                }
+            }
```

Comment:
In this case `runtimeChunk: { name: "vendor" }` makes sense.

Reply:
The input webpack config doesn't have `runtime` anywhere. How to identify when to apply `runtimeChunk`?

This is the input case for this output:
```js
module.exports = {
	entry: {
		app: './src/app.js',
		vendor: './src/vendors.js',
	},
	plugins: [
		new webpack.optimize.CommonsChunkPlugin({
			names: ["app", "vendor"],
			minChunks: 2
		})
	]
}
```